google/cloud/storage/blob.py

```diff
@@ -2166,11 +2132,9 @@ def _initiate_resumable_upload(
 
         upload_url = _add_query_parameters(base_url, name_value_pairs)
         upload = ResumableUpload(
-            upload_url, chunk_size, headers=headers, checksum=checksum
+            upload_url, chunk_size, headers=headers, checksum=checksum, retry=retry
```

Comment:
Curious will we be changing uploads to have unconditional retry by default as part of 3.0?

Reply:
Yes, my intention is to make sure that change happens before the release.